browser/main/SideNav/SideNav.styl

```diff
@@ -18,7 +18,37 @@
   display flex
   text-align center
 
+.extra-buttons
```

Comment:
The position doesn't feel right to me. 
<img src="https://user-images.githubusercontent.com/12984316/51013300-4cd6cf00-1594-11e9-87be-693c7503e34a.png" width="50%" />